cli/src/pcluster/validators/slurm_settings_validator.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+from enum import Enum
+
+from pcluster.validators.common import FailureLevel, Validator
+
+SLURM_SETTINGS_QUEUE_DENY_LIST = ["Nodes", "State", "PartitionName", "ResumeTimeout", "SuspendTime"]
+SLURM_SETTINGS_COMPUTE_RESOURCE_DENY_LIST = ["NodeName", "NodeHostName", "NodeAddr", "State", "Gres"]
```

Comment:
We have to double-check if the parameters in the `slurm.conf` are case-sensitive. If they aren't, we may have to do this validation case-insensitive.

Reply:
Fixed.
I also added a specific unit test.